src/SuspenseSubject.ts

```diff
@@ -84,7 +84,8 @@ export class SuspenseSubject<T> extends Subject<T> {
       clearTimeout(this._timeoutHandler);
     }
     this._innerSubscriber = this._innerObservable.subscribe(subscriber);
-    return this._innerSubscriber.add(this._reset);
+    // this._innerSubscriber.add(this._reset.bind(this));
```

Comment:
@jamesdaniels this was causing useSDK to throw out its value after it loaded. Would it be bad if I removed it?

Reply:
This could be important for other functions like `useUser`

Suggestion from James: have a flag `resetOnUnsubscribe` that's true for everything except the `useSDK` functions

Reply:
added a test for this, and it passes without adding `reset`, so I'm removing that line